### PR TITLE
Typos CI

### PR DESCRIPTION
**Pull request by egecetin**
Looks like codespell is missing too many errors. Maybe [typos](https://github.com/crate-ci/typos) can solve these

### CODE REVIEW

File: Packet++/header/SSLCommon.h [hunk 1]
```diff
@@ -388,7 +388,7 @@ namespace pcpp
 	/**
 	 * SSL/TLS symmetric encryption algorithms
 	 */
-	enum SSLSymetricEncryptionAlgorithm
+	enum SSLSymmetricEncryptionAlgorithm
```
Comment (seladb) [inline Packet++/header/SSLCommon.h:391]: enums are more difficult to clone and dperecate. I suggest we just leave it with the typo. What do you think?

File: Packet++/header/SSLHandshake.h [hunk 2]
```diff
@@ -656,7 +656,7 @@ class SSLServerHelloMessage : public SSLHandshakeMessage
 	/**
 	 * @return The size (in bytes) of all extensions data in this message. Extracted from the "extensions length" field
 	 */
-	uint16_t getExtensionsLenth() const;
+	uint16_t getExtensionsLength() const;
```
Comment (seladb) [inline Packet++/header/SSLHandshake.h:659]: ditto

File: Packet++/header/DhcpLayer.h [hunk 3]
```diff
@@ -688,7 +688,7 @@ namespace pcpp
 		 * @return DHCP message type as extracted from ::DHCPOPT_DHCP_MESSAGE_TYPE option. If this option doesn't exist the value of
 		 * ::DHCP_UNKNOWN_MSG_TYPE is returned
 		 */
-		DhcpMessageType getMesageType() const;
+		DhcpMessageType getMessageType() const;
```
[the remark this answers is not in the document]
Comment (egecetin) [reply Packet++/header/DhcpLayer.h:691]: I agree, I will update exception list and functions to deprecated 👍

File: Packet++/header/SSLCommon.h (hunk 1 above)
Comment (egecetin) [reply Packet++/header/SSLCommon.h:391]: We can duplicate values but probably not worth it 🤔. I think adding exception is easier.

**Review by seladb — APPROVED** [review]

**Pull request merged**